code/api/src/seeders/1-user.js

```diff
@@ -20,13 +21,17 @@ module.exports = {
         email: 'user@crate.com',
         password: bcrypt.hashSync('123456', config.saltRounds),
         role: params.user.roles.user,
+        style: 'Casual',
         createdAt: new Date(),
         updatedAt: new Date()
       }
     ])
   },
 
   down: (queryInterface, Sequelize) => {
-    return queryInterface.bulkDelete('users', null, {});
+    return (
+      queryInterface.bulkDelete('subscriptions', null, {}), //was causing foreign key error 
+      queryInterface.bulkDelete('users', null, {})
```

Comment:
In order to seed the database, I had to unseed it first. The original code failed because it was not deleting `subscriptions` and `subscriptions` without a user cannot exist. Simply delete the subscriptions first, then delete users.

Reply:
Makes sense, when exactly would this code run? when we close our server/start it up?

Reply:
Correct, you can see it if you look at `code/api/package.json` under `scripts/setup`